addons/event_sale/models/event.py

```diff
@@ -41,6 +43,18 @@ class Event(models.Model):
         'event.event.ticket', 'event_id', string='Event Ticket',
         copy=True)
 
+    sale_order_lines_ids = fields.One2many(
+        'sale.order.line', 'event_id',
+        string='All sale order lines pointing to this event')
+
+    sales_total_price = fields.Monetary(compute='_compute_sales_total_price')
+    currency_id = fields.Many2one(
+        'res.currency', string='Currency',
+        default=lambda self: self.env.company.currency_id.id, readonly=True)
+
+    start_sale_date = fields.Date('Start sale date', compute='_compute_start_sale_date')
+    event_tickets_available = fields.Boolean('Are tickets available', compute='_compute_event_tickets_available')
```

Comment:
Conditions used to allow registrations is a bit messy (even in standard code, not specific to this change). I would move this field in base event (see comment about _compute_event_tickets_available), and probably rename to event_registrations_open, something like that, and use it everywhere, notably in website event and event sale templates (instead of manually redoing the check on seats, dates and availability). That way we have the computation done in a single point (event), improve with tickets (event_sale) and used in all templates that require it.